recipes/dtale/meta.yaml

```
@@ -0,0 +1,55 @@
+{% set name = "dtale" %}
+{% set version = "1.7.3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 9fbfe214c577a2326b6c7dc1d8509f2c782b7728be790dd6d75ed718bae12010
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - python
+    - pip
+    - six
+  run:
+    - python
+    - lz4
+    - dash
+    - dash-daq
+    - Flask-Compress
+    - Flask
+    - future
+    - itsdangerous
+    - pandas
+    - requests
+    - scipy
+    - six
+
+test:
+  # Some packages might need a `test/commands` key to check CLI.
+  # List all the packages/modules that `run_test.py` imports.
+  imports:
+    - dtale
+    - dtale.cli
+
+about:
+  home: https://github.com/man-group/dtale/
+  license: LGPL
+  license_file: LICENSE
+  summary: 'D-Tale is the combination of a Flask back-end and a React front-end to bring you an easy way to view & analyze Pandas data structures'
+
+  # The remaining entries in this section are optional, but recommended.
+  doc_url: https://dtale.readthedocs.io
+  dev_url: https://github.com/man-group/dtale/
+
+extra:
+  recipe-maintainers:
+    - Juanlu001
+    - AlexS12
```

@AlexS12 Are you happy to be a maintainer of this?

As @aschonfeld offered to also be listed as a maintainer in https://github.com/man-group/dtale/pull/60#issuecomment-585521032 do you want to add them as well?:
```suggestion
    - AlexS12
    - aschonfeld
```

---

Oops, forgot! Applying